Source/WebCore/testing/Internals.mm

```diff
@@ -191,16 +191,21 @@ - (NSAttributedString *)_attributedStringForRange:(NSRange)range
 {
     auto device = adoptNS(MTLCreateSystemDefaultDevice());
 
+    UNUSED_PARAM(isWebGL2);
```

Comment:
This is OK to use even if it's actually used later?

Reply:
Yeah, it is. That expands to something like `(void)isWebGL2;`

It's a bit ugly to do it this way, but the alternative was to have it in two of the three branches.

Thanks @kenrussell